### DISCUSSION TITLE
240402

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#105] [팀회의]
## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
데이터 연동 완료(캘린더), 모달에 들어가는 시간 정하는 부분 진행중
## 오늘 할 일
모달에 들어가는 시간 정하는 부분 Ui완성, 네브바 데이터 연동, 모달 만들기
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항
event?.target.closest('dialog').close();로 닫지말고 그냥 이벤트 받은걸로 닫아주세요 필겸님

## 어제 한 일
내 히스토리 완성(프로필 편집 모달, 페이지네이션, 그룹 관리 모달)
팀 멤버 거의 완성
## 오늘 할 일
그룹 관리 모달 리퀘스트 끝내기
팀 멤버 완성하기
리스폰스 바뀐 거 맞춰서 signup 에러 풀기
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 팀/유저 메인에 들어가는 캘린더 height 절대적인 수치 말고 반응형으로 만들어주시면 좋을 것 같아요!
- 배포 관련

## 어제 한 일
- 모달 가져오는 코드 수정
- BoardSection 레이아웃 수정
- Drag&Drop 기능 구현

## 오늘 할 일
- Drag&Drop 기능 구현

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항
